cluster/gce/gci/configure-helper.sh

```diff
@@ -2244,6 +2244,7 @@ EOF
     local -r fluentd_gcp_yaml="${dst_dir}/fluentd-gcp/fluentd-gcp-ds.yaml"
     local -r fluentd_gcp_configmap_yaml="${dst_dir}/fluentd-gcp/fluentd-gcp-configmap.yaml"
     update-event-exporter ${event_exporter_yaml}
+    sed -i -e "s@{{ fluentd_gcp_version }}@${FLUENTD_GCP_VERSION}@g" "${fluentd_gcp_yaml}"
```

Comment:
I think there's a bug. If `FLUENTD_GCP_VERSION` is not registered (and it's not registered e.g. in GKE), script will crash with unbounded value

`${FLUENTD_GCP_VERSION}` -> `${FLUENTD_GCP_VERSION:-<some-default>}`